projects/framework-nuke/source/ftrack_framework_nuke/__init__.py

```diff
@@ -7,6 +7,11 @@
 from functools import partial
 import platform
 
+try:
```

Comment:
Comment from Mattias: 
[mlagergren](https://github.com/mlagergren) [2 weeks ago](https://github.com/ftrackhq/integrations/pull/540#discussion_r1658285091)
Why this change?